values.yaml

```diff
@@ -76,6 +77,9 @@ storage: filesystem
 secrets:
   haSharedSecret: ""
   htpasswd: ""
+  azure: {}
+  s3: {}
+  swift: {}
```

Comment:
The changes to these three lines seem spurious to me. Neither `azure` nor `swift` is referenced in this changeset, and `s3` is only referenced when `storage: s3`. While defaulting these to objects may remedy some other issue, in this PR they are spurious.